socs/agents/hwp_supervisor/agent.py

```diff
@@ -1696,9 +1715,9 @@ def pid_to_freq(self, session, params):
             }
         """
         if params['target_freq'] >= 0:
-            d = '0' if self.forward_is_cw else '1'
-        else:
             d = '1' if self.forward_is_cw else '0'
+        else:
+            d = '0' if self.forward_is_cw else '1'
```

Comment:
Sorry, why flip the relationship between 'forward' and positive frequency?

I had convinced myself last week that with `--forward-dir='cw'` on satp1/2, positive frequencies corresponded to 'cw' rotation. But this makes positive frequencies correspond to 'ccw', doesn't it?

Reply:
Sorry I said wrong statement last week, we noticed that this logic here has been wrong. 
Based on SO coords convention, position frequencies corresponds to 'ccw' seen from sky.
The doc string in pid_to_freq was correct but the logic was wrong.
https://github.com/simonsobs/socs/blob/94412e08a11a377a2fe6fbce42548f3fcfb32fdb/socs/agents/hwp_supervisor/agent.py#L1674-L1677